fido/Crypto/Fido2/Model/JavaScript/Encoding.hs

```diff
@@ -268,11 +242,11 @@ encodePublicKeyCredentialRequestOptions = encode
 -- | [(spec)](https://www.w3.org/TR/webauthn-2/#iface-pkcredential)
 -- Encodes the PublicKeyCredential for attestation, this function is mostly used in the tests where we emulate the
 -- of the client.
-encodeCreatedPublicKeyCredential :: M.PublicKeyCredential 'M.Create -> JS.CreatedPublicKeyCredential
+encodeCreatedPublicKeyCredential :: M.PublicKeyCredential 'M.Create 'True -> JS.CreatedPublicKeyCredential
 encodeCreatedPublicKeyCredential = encode
```

Comment:
It is rather unfortunate that we would have to first enter the `encodeRaw` function before using the function above. Would it be possible to also have this function accept `M.PublicKeyCredential 'M.Create 'True`?

Reply:
It would be, but then we'd have to involve singletons with `Bool`s, which doesn't yet seem that necessary